cocos/scripting/js-bindings/manual/network/XMLHTTPRequest.cpp

```diff
@@ -835,8 +856,6 @@ JS_BINDED_FUNC_IMPL(MinXmlHttpRequest, send)
                 return false;
             }
         }
-        else if(args.get(0).isNullOrUndefined()){
-        }
```

Comment:
Why to remove this? send() support to pass null since it's supported on Web too.